core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1326,6 +1326,14 @@ class ReplicaManager(val config: KafkaConfig,
             s"epoch ${leaderAndIsrRequest.controllerEpoch}")
         }
       val topicIds = leaderAndIsrRequest.topicIds()
+      def getTopicId(topicName: String): Option[Uuid] = {
```

Comment:
nit: how about `requestTopicId` instead of `getTopicId` so that it's clear that this comes from the LeaderAndIsr request?